public/Building_Tile_Example/Building_Tile_Example.py

```diff
@@ -1,7 +1,7 @@
 @fused.udf
-def udf(bounds: fused.types.Tile, table_path="s3://fused-asset/infra/building_msft_us"):
-    utils = fused.load(
-        "https://github.com/fusedio/udfs/tree/f928ee1/public/common/"
-    ).utils
-    df = utils.table_to_tile(bounds, table=table_path)
+def udf(bounds: fused.types.Bounds, table_path="s3://fused-asset/infra/building_msft_us"):
+    utils = fused.load("https://github.com/fusedio/udfs/tree/bb712a5/public/common/").utils
+    zoom = utils.estimate_zoom(bounds)
+    tile = utils.get_tiles(bounds, zoom=zoom)
+    df = utils.table_to_tile(tile, table=table_path)
```

Comment:
```suggestion
    df = utils.table_to_tile(bounds, table=table_path)
```

This probably doesn't work right now, but I think we should first update `table_to_tile` to take bounds directly, instead of having to update all UDFs that use this to convert the bounds into a box or tiles

Reply:
Hmm, I think this can be done in a separate PR (or atleast after the main task is completed)

Reply:
Agreed, make new ticket please :)

Reply:
I would strongly argue we should do that first. If we want to use `bounds` for all those UDFs, then our helper functions should be ready for doing that, otherwise we introduce churn and show users the convoluted solution

Reply:
I get your point @jorisvandenbossche but I'd like to first get this to a point where it's working everywhere with `bounds` and then we can improve / optimize at later time

So:
1. Update all public UDFs to just use `bounds` now, even if not perfect -> New users only see `bounds` ASAP
2. Update helpful functions over time
3. Improve Public UDFs as we roll out new helper functions